app/src/main/java/eu/kanade/tachiyomi/source/online/english/Mangadex.kt

```diff
@@ -800,14 +801,18 @@ open class Mangadex(override val lang: String, private val internalLang: String,
                 Pair("Thai", "32"),
                 Pair("Filipino", "34"))
 
-        private val FOLLOW_STATUS_LIST =
-                listOf(
-                        Triple("Unfollowed", 0, SManga.FollowStatus.UNFOLLOWED),
-                        Triple("Reading", 1, SManga.FollowStatus.READING),
-                        Triple("Completed", 2, SManga.FollowStatus.COMPLETED),
-                        Triple("On hold", 3, SManga.FollowStatus.ON_HOLD),
-                        Triple("Plan to read", 4, SManga.FollowStatus.PLAN_TO_READ),
-                        Triple("Dropped", 5, SManga.FollowStatus.DROPPED),
-                        Triple("Re-reading", 6, SManga.FollowStatus.RE_READING))
+        private val FOLLOW_STATUS_LIST = listOf(
```

Comment:
I wanted to use `private data class Mapping(val id: Int, val followStatus: FollowStatus, val string: String)` instead of `Triple` but decided against it